src/root/real_t.d

```diff
@@ -0,0 +1,119 @@
+// Compiler implementation of the D programming language
+// Copyright (c) 1999-2016 by Digital Mars
+// All Rights Reserved
+// written by Walter Bright
+// http://www.digitalmars.com
+// Distributed under the Boost Software License, Version 1.0.
+// http://www.boost.org/LICENSE_1_0.txt
+
+module ddmd.root.real_t;
+
+static import core.math, core.stdc.math;
+import core.stdc.errno;
+import core.stdc.stdio;
+import core.stdc.stdlib;
+import core.stdc.string;
+
+alias real_t = real;
+
+private
+{
+    version(CRuntime_DigitalMars) __gshared extern (C) extern const(char)* __locale_decpoint;
+
+    version(CRuntime_Microsoft) extern (C++)
+    {
+        struct longdouble { real_t r; }
+        size_t ld_sprint(char* str, int fmt, longdouble x);
+        longdouble strtold_dm(const(char)* p, char** endp);
+    }
+}
+
+extern (C++) struct TargetReal
+{
+    static __gshared real_t snan;
+
+    static this()
```

Comment:
At what point is this expected to be executed?  (Hint, GDC replaces the ``PortInitializer`` construct with ``Port::init`` because this information can only be initialized after the backend has started).